src/app/Menu/MenuPost.js

```diff
@@ -11,15 +12,30 @@ const MenuPost = ({
   reblog,
   isReblogged,
   canReblog,
-  openCommentingDraft,
   isPostLiked,
   isPostDisliked,
   likePost,
   unlikePost,
   dislikePost,
   content,
   onEdit,
+  modal
 }) => {
+  const handleReplyButtonClick = (e) => {
+    e.stopPropagation();
+    if (modal) {
+      animateScroll.scrollMore(400, { containerId: 'busyModal' });
```

Comment:
@adcpm right I can calculate the exact position but we cannot have smooth scroll with simple anchor (hashtag) links. It's a good library and can do many things for us including go back to top/bottom features in timeline.